src/llm_vm/completion/models.py

```diff
@@ -0,0 +1,22 @@
+import llm_vm.onsite_llm as llms
+import json
+import os
+
+MODEL_DICT = {
+    "opt":llms.Small_Local_OPT,
+    "bloom":llms.Small_Local_Bloom,
+    "neo":llms.Small_Local_Neo,
+    "llama":llms.Small_Local_LLama,
+    "gpt":llms.GPT3,
+    "chat_gpt":llms.Chat_GPT
+}
+
```

Comment:
thats fine for now, we WILL need to do better search path wise, i can hack that out for us easily

Reply:
What does this mean?

Reply:
"where does the user put the config so we find it", theres some nice standardized ways To handle that